config/environments/development.rb

```diff
@@ -61,4 +61,13 @@
 
   # Mailer for Devise?
   # config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => Rails.application.secrets.mailer_user_name,
```

Comment:
Use the new syntax. Eg.:  `user_name: Rails.application.secrets.mailer_user_name`

Reply:
Done!